site/specs/numbers.json

```diff
@@ -7661,8 +7661,9 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieve a list of the TelephoneNumbersAssignment orders that are associated with the account. <p>A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.</p></p>\n",
-        "operationId": "GET /accounts/{accountId}/numbersAssignment",
+        "description": "Retrieve a list of the TelephoneNumbersAssignment orders that are associated with the account. A maximum of 1,000 orders can be retrieved per request. If no date range or specific query parameter (marked by <b class=\"required\">*</b> below) is provided, the order results will be limited to the last two years.",
```

Comment:
In the new world we aren't marking things as required with a `*`.  Here's what it looks like now
<img width="661" alt="image" src="https://user-images.githubusercontent.com/2125458/164008298-3b9f7586-8ecf-4155-a31d-cafa39347cf8.png">